packages/@guardian/source-foundations/src/typography/typography.stories.mdx

```diff
@@ -61,48 +74,39 @@ typography API assigns font size in rems.
 font-family: 'GH Guardian Headline, Guardian Egyptian Web, Georgia, serif';
 ```
 
--   `headline.xxxsmall()` -> 17px
--   `headline.xxsmall()` -> 20px
--   `headline.xsmall()` -> 24px
--   `headline.small()` -> 28px
--   `headline.medium()` -> 34px
--   `headline.large()` -> 42px
--   `headline.xlarge()` -> 50px
+<Story name="headline">
+	<FontStylesRenderer fontName="headline" fontStyles={headlineObjectStyles} />
+</Story>
 
 ### Body
 
 ```css
 font-family: 'GuardianTextEgyptian, Guardian Text Egyptian Web, Georgia, serif';
 ```
 
--   `body.xsmall()` -> 14px
--   `body.small()` -> 15px
--   `body.medium()` -> 17px
+<Story name="body">
+	<FontStylesRenderer fontName="body" fontStyles={bodyObjectStyles} />
+</Story>
 
 ### Text sans
 
 ```css
-font-family: 'GuardianTextSans, Guardian Text Sans Web, Helvetica Neue, Helvetica, Arial, Lucida Grande, sans-serif';
+font-family: 'GuardianTextSans, Guardian Text Sans Web, Helvetica Neue, Helvetica, Arial, Lucnamea Grande, sans-serif';
```

Comment:
is `Lucnamea` an intentional name change?

Reply:
Great spot 👌